src/main/config/proxy-compose.yaml.ctmpl

```diff
@@ -8,17 +8,10 @@ proxy:
   {{.Data.proxy_volumes}}
   {{.Data.log_driver}}
   environment:
-    AUTH_TYPE: AuthType None
```

Comment:
locally yes, but should test again in dev

On Monday, November 23, 2015, abaumann <notifications@github.com> wrote:

> In src/main/config/proxy-compose.yaml.ctmpl
> <https://github.com/broadinstitute/rawls/pull/229#discussion_r45610540>:
>
> > @@ -8,17 +8,10 @@ proxy:
> >    {{.Data.proxy_volumes}}
> >    {{.Data.log_driver}}
> >    environment:
> > -    AUTH_TYPE: AuthType None
>
> did we verify that it works with these removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/rawls/pull/229/files#r45610540>.
>